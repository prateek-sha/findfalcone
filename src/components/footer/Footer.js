import React from "react";
import { Grid, Typography, makeStyles } from "@material-ui/core";
import Fade from "react-reveal/Fade";

const useStyles = makeStyles({
  root: {
    margin: "2em",
    color: "white",
    textAlign: "center",
  },
  link: {
    color: "white",
    textDecoration: "none",
    cursor: "pointer",
  },
});

export const Footer = () => {
  const classes = useStyles();
  return (
    <Grid className={classes.root}>
      <Fade right>
        <Typography>
          Coding problem -
          <a href="www.geektrust.in/finding-falcone" className={classes.link}>
            www.geektrust.in/finding-falcone
          </a>
        </Typography>
      </Fade>
    </Grid>
  );
};
