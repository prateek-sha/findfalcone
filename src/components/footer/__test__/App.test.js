import React from "react";
import { Footer } from "../Footer";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

it("matches snapshot ", () => {
  const tree = renderer.create(<Footer></Footer>).toJSON();
  expect(tree).toMatchSnapshot();
});
