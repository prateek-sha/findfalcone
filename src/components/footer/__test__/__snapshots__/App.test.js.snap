// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot  1`] = `
<div
  className="MuiGrid-root makeStyles-root-1"
>
  <div
    className="react-reveal"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      Coding problem -
      <a
        className="makeStyles-link-2"
        href="www.geektrust.in/finding-falcone"
      >
        www.geektrust.in/finding-falcone
      </a>
    </p>
  </div>
</div>
`;
